src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -619,8 +634,40 @@ private void TestRunCompleteHandler(object sender, TestRunCompleteEventArgs e)
         {
             // Stop the progress indicator as we are about to print the summary
             this.progressIndicator?.Stop();
+            var testsPassed = 0;
+            var testsFailed = 0;
+            var testsSkipped = 0;
+            var testsTotal = 0;
             Output.WriteLine(string.Empty, OutputLevel.Information);
 
+            foreach (var sd in this.sourceSummaryDictionary.ToArray())
+            {
+                var summary = this.sourceSummaryDictionary[sd.Key];
+                testsPassed += summary.PassedTests;
+                testsFailed += summary.FailedTests;
+                testsSkipped += summary.SkippedTests;
+                testsTotal += summary.TotalTests;
+
+                if (verbosityLevel == Verbosity.Quiet)
+                {
+                    if (summary.FailedTests > 0)
+                    {
+                        // Failed! Pass {1} Failed {2} skipped {3} Time : 233 se ({4})
+                        Output.Information(false, ConsoleColor.Red, string.Format(CultureInfo.CurrentCulture, CommandLineResources.TestRunSummaryFailed, summary.TotalTests, summary.PassedTests, summary.FailedTests, summary.SkippedTests, GetFormattedDurationString(summary.TimeSpan), sd.Key.Split('\\').Last(), targetFramework));
```

Comment:
1. What happens when target framework is not there? Do we always have a target framework?
2. Directly appending target framework will append the full name of target framework. eg: .NetCoreApp, Version 3.0.
Do we want to consider appending short folder name (netcoreapp3.0) as suggested by @adamralph ?
cc: @singhsarab